### DISCUSSION TITLE
How to change Language under Windows ?

### DISCUSSION THREAD

**Discussion by ProTechEx** [flameshot-org/flameshot#3513] [Q&A]
Hey, 

I am a new user of Flameshot. I quickly fell in love with this wonderful tool :)
However, I can't find the option to change the system language in the settings..

Am I just overlooking it or is the option not there at all?

**Comment by GaborElAyoubi** [comment]
As far as I know it is only available in English

**Comment by basgroot** [comment]
After moving to a new desktop my Flameshot (v13.1.0, win11) appeared in Dutch. There seems to be no option to change to English. Or am I overlooking something?
<img width="184" height="294" alt="image" src="https://github.com/user-attachments/assets/f83b85ec-3226-4831-9749-ba772678018f" />

**Reply by basgroot** [reply]
Workaround: https://github.com/flameshot-org/translation-instruction/issues/25

**Comment by ElTh0r0** [comment]
I provided a PR to fix this: #4249
